boards.txt

```diff
@@ -350,3 +423,58 @@ OSHChip.menu.softdevice.s130.softdeviceversion=2.0.1
 OSHChip.menu.softdevice.s130.upload.maximum_size=151552
 OSHChip.menu.softdevice.s130.build.extra_flags=-DNRF51 -DS130 -DNRF51_S130
 OSHChip.menu.softdevice.s130.build.ldscript=armgcc_s130_nrf51822_xxac.ld
+
+
+PCA1000X.name=PCA1000X
```

Comment:
Not sure if I asked this before, is this a Nordic board? If so it would be nice to give it a nicer name.

One more request, could you please also update the read me with links. After that we should be good to merge.